extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -360,6 +360,11 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
         expect(ampAnalyticsElement.innerHTML.indexOf(url)).to.not.equal(-1);
       });
     });
+    it('centers iframe in slot', () => {
```

Comment:
The centering happens irrespective of the height/width values, so is that really necessary? I don't mind adding them; I'm just wondering.